### PR TITLE
Rebase to 3.18

### PR DESCRIPTION
**Pull request by thespad**
[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

------------------------------

 - [x] I have read the [contributing](https://github.com/linuxserver/docker-minetest/blob/master/.github/CONTRIBUTING.md) guideline and understand that I have made the correct modifications

------------------------------

### CODE REVIEW

**Comment by thespad** [comment]
@KingBBQ this should be a very straightforward change, but would appreciate you giving it a once-over when the PR build is complete.

**Review by aptalca — APPROVED** [review]

**Comment by thespad** [comment]
Won't build at the moment ecause irrlicht committed https://github.com/minetest/irrlicht/commit/c40045a40acf87c2153637b94eb03c0c32c0edd6 and released against it, and while minetest has committed https://github.com/minetest/minetest/commit/6bf63d4b418b49f7aa7b76bd56228ac07d6e9318 there's no release with it so the build fails.

**Comment by thespad** [comment]
Upstream released 5.8.0 so this should now build

**Comment by thespad** [comment]
Ugh, no 5.8.0 tagged release for https://github.com/minetest/minetest_game so the build is failing

**Review by aptalca — APPROVED** [review]

File: Jenkinsfile [hunk 1]
```diff
@@ -32,7 +32,7 @@ pipeline {
     CI_PORT='80'
     CI_SSL='false'
     CI_DELAY='120'
-    CI_DOCKERENV='TZ=Europe/London|CLI_AGRS="--gameid devtest"'
```
Comment (zJudGenie) [inline Jenkinsfile]: Is CLI_AGRS a typo ? Maybe it should be CLI_ARGS

File: jenkins-vars.yml [hunk 2]
```diff
@@ -23,6 +23,6 @@ repo_vars:
   - CI_PORT='80'
   - CI_SSL='false'
   - CI_DELAY='120'
-  - CI_DOCKERENV='TZ=Europe/London|CLI_AGRS="--gameid devtest"'
```
Comment (zJudGenie) [inline jenkins-vars.yml]: Same typo here I think

**Review by zJudGenie — COMMENTED** [review]
Writing typo, also why the gameid is set to devtest ? Just to test it ?

File: Jenkinsfile (hunk 1 above)
Comment (aptalca) [reply Jenkinsfile]: We had multiple sets of eyes miss it the first time around lol

**Pull request merged**